### ISSUE TITLE
Train html loss inf

### ISSUE THREAD

**Issue by holyfisher** [poloclub/unitable#45]
During HTML training, somethings the loss occasionally becomes inf, causing the total loss to remain inf after accumulation. Why doesn't gradient clipping take effect, and what is causing this issue?
